etc/lmod-setup.csh

```diff
@@ -36,11 +36,7 @@ else if ( "$L_MACHINE" == singularity ) then
    module purge
 
 else if ( "$L_MACHINE" == gaea ) then
-   set ENV="/lustre/f2/dev/role.epic/contrib/apps/lmod/lmod/init/csh"
-   source $ENV
-
-   setenv LMOD_SYSTEM_DEFAULT_MODULES "modules/3.2.11.4"
-   module --initial_load --no_redirect restore
+   source /lustre/f2/dev/role.epic/contrib/Lmod_init.csh
```

Comment:
While using `run_WE2E_tests.sh` to run the fundamental WE2E tests, I see the following:

```
(regional_workflow) gaea13 tests/WE2E> ./run_WE2E_tests.sh test_type="fundamental" machine="gaea" account="nggps_emc"
/lustre/f2/dev/role.epic/contrib/apps/lmod/lmod/init/csh: line 2: setenv: command not found
/lustre/f2/dev/role.epic/contrib/apps/lmod/lmod/init/csh: line 3: setenv: command not found
/lustre/f2/dev/role.epic/contrib/apps/lmod/lmod/init/csh: line 4: setenv: command not found
/lustre/f2/dev/role.epic/contrib/apps/lmod/lmod/init/csh: line 5: setenv: command not found
/lustre/f2/dev/role.epic/contrib/apps/lmod/lmod/init/csh: line 6: setenv: command not found
/lustre/f2/dev/role.epic/contrib/apps/lmod/lmod/init/csh: line 7: setenv: command not found
/lustre/f2/dev/role.epic/contrib/apps/lmod/lmod/init/csh: line 8: setenv: command not found
/lustre/f2/dev/role.epic/contrib/apps/lmod/lmod/init/csh: line 63: syntax error: unexpected end of file
```
Ultimately, the tests are configured and run, but is it possible to address the warning/error messages from `/lustre/f2/dev/role.epic/contrib/apps/lmod/lmod/init/csh`?

Is this being caused because the `csh` script is being loaded, then `bash` scripts are being run?  I should say that I don't see this happening in the current `develop` branch.

It should also be noted that this behavior was also seen when attempting to build the SRW using `devbuild.sh` after using:

`source etc/lmod-setup.csh gaea`

Reply:
@MichaelLueken @danielabdi-noaa - 
The solution that seems to work is that  `lmod-setup.sh` involves calling the `/lustre/f2/dev/role.epic/contrib/Lmod_init.sh`. 
When csh is used,  the `lmod-setup.csh` remains the same as is currently in the develop branch, and involves sourcing `/lustre/f2/dev/role.epic/contrib/apps/lmod/lmod/init/csh`. So the user with csh would run `source ./etc/lmod-setup.csh` This works successfully, because the lmod-setup.sh is called later from build script regardless of the shell.

With these changes, I was able to build the SRW successfully using ./devbuild.sh approach and cmake approach

Reply:
Following your modification to use what is currently in develop, the SRW App builds without issue and WE2E test generation occurs without error messages.  Thank you very much for taking care of this!